spec/v2/index.bs

```diff
@@ -454,7 +454,7 @@ Advisement:
         <td>String
         <td>O*
         <td>If present, emissions resulting from recent (i.e., previous 20 years) carbon stock loss due to land conversion directly on the area of land under consideration. The value of this property MUST include direct land use change (dLUC) where available, otherwise statistical land use change (sLUC) can be used.
-          The value MUST be calculated per <{CarbonFootprint/declaredUnit|declared unit}> with unit `kg of CO2 equivalent per declared unit` (`kgCO2e / declaredUnit`), expressed as a [=decimal=] equal to or greater than zero.
+          The value MUST be calculated per <{CarbonFootprint/declaredUnit|declared unit}> with unit `kg per declared unit` (`kg / declaredUnit`), expressed as a [=decimal=] equal to or greater than zero.
```

Comment:
Martin mentioned this to me a few days ago, I think we realized that indeed instead of declaredUnit in the denominator, the correct property would be unitaryProductAmount, which would be in the unit of declaredUnit. But would need his eyes here to double check in case I misunderstood.

Reply:
Hmm, now I am doubting whether I am correctly reviewing this. Because now I realize this PR is made to the property dLucGhgEmissions, but as I understood per the task listed https://www.notion.so/sinefoundation/Documentation-Fixes-to-Tech-Specs-v2-d5229a4132e3425d931995b136613006 we should be updating the field biogenicCarbonContent, i.e. the property above. And indeed that should not reference kg of CO2 equivalent, but rather just kg of CO2. Perhaps I have completely misunderstood however...

Reply:
Indeed. I'll revise this immediately.